### PR TITLE
NCBD-463 Ticker external agent actions

### PR DESCRIPTION
**Pull request by raycar5**

### CODE REVIEW

File: src/mappings/entities/sto.ts [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import { Codec } from "@polkadot/types/types";
+import { SubstrateEvent } from "@subql/types";
```
Comment (polymath-eric) [inline src/mappings/entities/sto.ts:2]: (nit) lint errors

File: schema.graphql [hunk 2]
```diff
@@ -97,3 +97,8 @@ type FundingEvent @entity {
   value: String!
   total_issued_in_funding_round: String!
 }
+
```
Comment (polymath-eric) [inline schema.graphql:100]: Don't we need to add external agents stuff to the schema here?

It looks like tooling has stuff for queries on ExternalAgents.

Comment (raycar5) [reply schema.graphql:100]: It's already in the branch, TickerExternalAgentAction, expand the diff a bit up to line 81

Comment (raycar5) [reply schema.graphql:100]: It's already in the branch, TickerExternalAgentAction, expand the diff a bit up to line 81

Comment (polymath-eric) [reply schema.graphql:100]: My bad, I assumed it would be in this one.

**Review by polymath-eric — APPROVED** [review]
LGTM

**Pull request merged**